src/libs/antares/study/scenario-builder/rules.h

```diff
@@ -148,6 +148,7 @@ class Rules final : private Yuni::NonCopyable<Rules>
     Data::AreaLink* getLink(const AreaName& fromAreaName,
                             const AreaName& toAreaName,
                             bool updaterMode);
+    bool checkGroupExists(std::string& group_name);
```

Comment:
Constness police ;) : argument should be const ref (and method const, probably).

Reply:
done